### PR TITLE
Change the xcode version for release carthage job

### PR DESCRIPTION
**Pull request by sunchunqiang**
Change the xcode version from 10.0.0 to 9.2.0 for release carthage job to fix the carthage build timeout issue

### CODE REVIEW

File: CreateCarthageArchive.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# carthage archive | tee "carthageout.txt"
-frameworkfilename=$(carthage archive |  grep -o 'Created .*\.zip$'  | grep -o   '\S*\.zip$')
-
+carthage archive | tee "carthageout.txt"
+ls *.zip
```
Comment (muellerfr) [inline CreateCarthageArchive.sh]: why are you running "ls" here?

File: .circleci/config.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ jobs:
   release_carthage:
     # Specify the Xcode version to use
     macos:
-      xcode: "10.0.0"
+      xcode: "9.2.0"
```
Comment (muellerfr) [inline .circleci/config.yml]: How does changing the xcode version help?

Comment (palpatim) [reply .circleci/config.yml]: Xcode 9.3 [isn't packaged with `docsetutil`](https://github.com/tomaz/appledoc/issues/628), which we use for generating appledoc. We decided to force-downgrade the Xcode version we're building with.

Comment (rohandubal) [reply .circleci/config.yml]: @palpatim We already made the change for that for our docs workflow, but this one would also use the older xcode for our Carthage builds.

File: CreateCarthageArchive.sh (hunk 1 above)
Comment (rohandubal) [reply CreateCarthageArchive.sh]: I believe this is for debugging purposes.

**Pull request merged**